firmware/tunerstudio/rusefi.input

```diff
@@ -431,6 +431,14 @@ fileVersion = { 20171101 }
       yBins       = cltIdleCorr
       gauge       = CLTGauge
 
+	curve = iacCoastingCurve, "Coasting IAC Position for Auto-Idle"
+      columnLabel = "Coolant", "Multiplier"
+      xAxis       = -40, 120, 10
+      yAxis       =  0,  100, 10
+      xBins       = iacCoastingBins, coolant
+      yBins       = iacCoasting
+      gauge       = idleAirValvePositionGauge
+
```

Comment:
We set gauge to IAC position for convenience - it helps to grab some real IAC values from the idling engine and put into the table.